Monika After Story/game/script-topics.rpy

```diff
@@ -3641,13 +3641,14 @@ label monika_trolley:
     m 1eua "Oh, cool. I love thinking about these sorts of thought experiments."
     m 1euc "I guess we're talking about real people, right? I wouldn't have a particular preference if I knew they didn't have free will."
     m 1dsc "Hmmm..."
-    m 1eua "If we were talking about the classic problem, of course I would switch."
-    m 1rkc "But if we're talking about real people..."
-    m 3esc "I wouldn't switch."
-    m 4esc "That's the only logical choice, if you think about it."
-    m 1euc "It feels nice to say that you'd save the most people, doesn't it? But people only say that because nobody in the example matters to them."
-    m "If they actually cared about the person, they'd kill the other five in a heartbeat."
-    m 2tsb "Do you know anyone who would want a self-driving car to save other people by killing the driver? Didn't think so, [player]."
+    m 3esa "The classic trolley problem has us choose between letting the trolley run over 5 people, or switching to the track where only 1 person will be killed."
+    m 1esb "I think this problem is too easy. Of course I would switch."
```

Comment:
I might not say that it's too easy, or at least say that there might be more to it. If it was too easy, and everyone had the same obvious answer, it wouldn't be such a famous question.

Reply:
I suppose the phrasing here is kinda poor. I'd interpret this line more as Monika saying it's easy for her personally to answer, not necessarily in general. I feel like she's been quite clearly shown to be on the ends-justify-the-means side of the spectrum. There might be a bit of a personal bias showing here in that the choice seems almost trivially easy. As someone who'd pull the lever as well, I simply don't understand what kind of a mindset you'd need to have to choose otherwise, short of simply not caring.

Reply:
I interpreted the line as Monika saying that the question is incredibly easy, and the answer is obvious to all. True, Monika would likely hold the stance to switch, but the way it's worded sounds like she's saying it's an easy question to begin with, with an obvious one answer.

Reply:
I think an easy fix would be to just remove that "too". 

Easy for Monika, but not so, so easy and obvious, that there's an expectation for everyone.